src/main/java/gregtech/common/covers/filter/OreDictionaryItemFilter.java

```diff
@@ -29,7 +30,7 @@ public class OreDictionaryItemFilter extends ItemFilter {
 
     private final List<OreDictExprFilter.MatchRule> matchRules = new ArrayList<>();
     private static final Hash.Strategy<ItemStack> strategy = ItemStackHashStrategy.builder().compareItem(true).compareDamage(true).build();
-    private final Object2BooleanOpenCustomHashMap<ItemStack> recentlyChecked = new Object2BooleanOpenCustomHashMap<>(strategy);
+    private final Map<ItemStack, Boolean> recentlyChecked = new Object2BooleanOpenCustomHashMap<>(strategy);
```

Comment:
`Set<ItemStack>`?

Reply:
It looks like this does actually have a case for `stack, false` in the map, so i think we need to keep it as a map